src/App.css

```diff
@@ -28,11 +30,140 @@
   color: #61dafb;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+body {
```

Comment:
is this the same thing as this?
`html {
    /* 1rem = 20px */
    font-size: 125%;
}`